internal/rsl/rsl.go

```diff
@@ -77,34 +80,38 @@ type Entry struct {
 	TargetID plumbing.Hash
 }
 
-// NewEntry returns an Entry object for a normal RSL entry.
-func NewEntry(refName string, targetID plumbing.Hash) *Entry {
-	return &Entry{RefName: refName, TargetID: targetID}
+// NewStandardEntry returns a StandardEntry object for a normal RSL entry.
+func NewStandardEntry(refName string, targetID plumbing.Hash) *StandardEntry {
```

Comment:
bikeshedding a little bit here since we're renaming anyway: If it's not a StandardEntry, does that make anything else a non-standard Entry? 🤔 

Should this be a `ReferenceEntry` / `ReferenceStateEntry` / something else?

Reply:
Fair point, I like ReferenceEntry the best. I can submit it fairly quickly.

Reply:
Done!